tests/readConcern/readconcern-ctor-001.phpt

```diff
@@ -5,8 +5,6 @@ MongoDB\Driver\ReadConcern construction
 
 var_dump(new MongoDB\Driver\ReadConcern());
 var_dump(new MongoDB\Driver\ReadConcern(null));
-var_dump(new MongoDB\Driver\ReadConcern(MongoDB\Driver\ReadConcern::LOCAL));
```

Comment:
Please keep one example using a defined constant, as you did in other tests.

Reply:
done